### DISCUSSION TITLE
Appropriate citation

### DISCUSSION THREAD

**Discussion by kunc** [PyO3/maturin#1030] [Q&A]
Hi,
I like using the _maturin_ in a scientific environment and I would love to give the credit properly - what is the proper way to cite the maturin package? Is there any paper I can refer to in my work?
Thanks

**Comment by messense** [comment]
I don't know, perhaps @konstin can help answer this.

**Reply by konstin** [reply]
Unfortunately we don't have any publication. I thought about writing something up because i've seen that we quite a number of scientific users but that would be a larger project; right know i'd recommend looking what your citation style says about citing software/github projects, they normally have some examples for that.
